src/app/public/testing/dropdown/dropdown-fixture.ts

```diff
@@ -0,0 +1,196 @@
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  SkyDropdownComponent
+} from '../../modules/dropdown/dropdown.component';
+
+import {
+  SkyPopoversFixtureDropdownItem
+} from './popovers-fixture-dropdown-item';
+
+import {
+  SkyPopoversFixtureDropdownMenu
+} from './popovers-fixture-dropdown-menu';
+
+import {
+  SkyPopoversFixtureDropdown
+} from './popovers-fixture-dropdown';
+
+/**
+ * Allows interaction with a SKY UX dropdown component just as a user would.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyDropdownFixture {
+
+  private debugEl: DebugElement;
+
+  /**
+   * Retrieves information about the dropdown component.
+   */
+  public get dropdown(): SkyPopoversFixtureDropdown {{
+    const button = this.buttonDebugElement;
+
+    if (!button) {
+      return;
+    }
+
+    const buttonCmp = button.componentInstance as SkyDropdownComponent;
```

Comment:
I wouldn't expect to see the component itself being imported into the fixture. What we typically do is pick values off the DOM elements; that way we get assurance that the DOM itself is in the expected state. Also, I'm not sure what the ramifications of importing a component into a fixture is in terms of packaging the test fixtures, since they essentially are their own bundle.

Reply:
The component has been removed and the properties have been updated to return things via the DOM attributes.